metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_v2.py

```diff
@@ -1014,3 +1037,22 @@ def inspect_session_metadata(self, conn: SnowflakeConnection) -> None:
     # Stateful Ingestion Overrides.
     def get_platform_instance_id(self) -> str:
         return self.config.get_account()
+
+    # Ideally we do not want null values in sample data for a column.
+    # However that would require separate query per column and
+    # that would be expensive, hence not done.
+    def get_sample_values_for_table(self, conn, table_name, schema_name, db_name):
```

Comment:
does this repeat work that the profiler does?

Reply:
Kind of, but not exactly. With current implementation(with GE), profile can get only limited number of sample values(upto 20).